mavros_extras/src/plugins/odom.cpp

```diff
@@ -198,35 +103,24 @@ class OdometryPlugin : public plugin::PluginBase {
 	 * @brief Handle ODOMETRY MAVlink message.
 	 *
 	 * Message specification: https://mavlink.io/en/messages/common.html#ODOMETRY
+	 *
+	 * Callback for mavlink ODOMETRY messages sent from the FCU. It will be received with parent_frame
+	 * "local_origin_ned" (aka "map_ned") and child_frame "fcu_frd" (aka "base_link_frd"). We will publish it
+	 * with the parent and child frame being set in the rosparams "odometryfcu/odom_*_id_des" (set in px4_config.yaml).
```

Comment:
Comment seems to be outdated. I propose:


Callback for mavlink ODOMETRY messages sent from the FCU. According to the mavlink specification, all quantities are for the child frame (fcu_frd), expressed in the parent frame (local_origin_ned). To be compliant with ROS REP 147 for the published nav_msgs/Odometry, the data will be appropriately transformed and published. Frames for the publish message should be specified in specified in the rosparams "odometry/fcu/odom_*_id_des" (set in px4_config.yaml).